ui/pages/confirmations/components/confirm/dapp-swap-comparison-banner/dapp-swap-comparison-banner.tsx

```diff
@@ -155,41 +113,38 @@ const DappSwapComparisonInner = () => {
 
   return (
     <Box>
-      <Box
-        borderColor={BoxBorderColor.BorderMuted}
-        borderWidth={1}
-        className="dapp-swap_wrapper"
-        marginBottom={4}
-        marginTop={2}
-        padding={1}
-      >
-        <SwapButton
-          className="dapp-swap_dapp-swap-button"
-          type={
-            selectedSwapType === SwapType.Current
-              ? SwapButtonType.ButtonType
-              : SwapButtonType.Text
+      <Tabs
+        defaultActiveTabKey="marketRate"
+        onTabClick={(tabKey) => {
+          if (tabKey === 'marketRate') {
+            updateSwapToCurrent();
+          } else if (tabKey === 'mmswap') {
+            updateSwapToSelectedQuote();
           }
-          onClick={updateSwapToCurrent}
-          label={t('marketRate')}
```

Comment:
Declaring method inline here is avoidable I think, to reduce possible re-rendering.